### PR TITLE
Change some settings' name 

### PR DESCRIPTION
**Pull request by Asnelchristian**

### CODE REVIEW

File: bears/ruby/RuboCopBear.py [hunk 1]
```diff
@@ -60,9 +62,9 @@ def generate_config(filename, file,
                                                  "REVIEW"),
                         min_if_unless_guard: int=1,
                         indent_size: int=2,
-                        method_name_case: str='snake_case',
+                        method_naming_convention: str='snake_case',
```
Comment (sils1297) [inline bears/ruby/RuboCopBear.py]: why are we still doing snake_case and not snake?

File: bears/ruby/RuboCopBear.py [hunk 2]
```diff
@@ -9,6 +9,22 @@
...
+
+     >>> map_naming_convention_to_rubocop('snake')
+     'snake_case'
+     >>> map_naming_convention_to_rubocop('camel')
+     'camelCase'
+     '''
+    if setting == 'camel':
+        return 'camelCase'
+    else:
+        return 'snake_case'
```
Comment (sils1297) [inline bears/ruby/RuboCopBear.py]: nah you should check for more and error out properly if it's neither camel or snake IMO

File: bears/ruby/RuboCopBear.py (hunk 1 above)
Comment (Adrianzatreanu) [inline bears/ruby/RuboCopBear.py]: agreed, this should be 'snake' instead, not 'snake_case' :p

if someone asks you: what case would you like to use? 'snake'! not 'snake_case'! :p

**Comment by Adrianzatreanu** [comment]
still bumping for you

File: bears/ruby/RuboCopBear.py (hunk 1 above)
Comment (Makman2) [inline bears/ruby/RuboCopBear.py]: though `snake-case` is easier to read in a coafile context^^

**Comment by sils** [comment]
ack https://github.com/coala/coala-bears/pull/799/commits/27a01f7889a0be12b7ffcee3a03a81b4eb2a4768

File: bears/ruby/RuboCopBear.py [hunk 3]
```diff
@@ -9,6 +9,16 @@
 from coalib.results.Result import Result
 
 
+def map_naming_convention_to_rubocop(setting):
+    '''
+    This function converts the string ``setting`` (that refers to a naming
+    convention) into a ``rubocop`` specific naming convention.
+    '''
+    naming_convention = {'camel': 'camelCase', 'snake': 'snake_case'}
```
Comment (sils) [inline bears/ruby/RuboCopBear.py]: no need for a function, just use `naming_convention.get(setting, "snake_case")` and dump this function

**Comment by sils** [comment]
unack https://github.com/coala/coala-bears/pull/799/commits/37546a5897c0d831f8b84acf8db49eac681962d6

**Comment by sils** [comment]
reack https://github.com/coala/coala-bears/pull/799/commits/3f6eaa9a7ea17f7b7587d44dccdcfedad610304f

File: bears/ruby/RuboCopBear.py [hunk 4]
```diff
@@ -62,9 +62,9 @@ def generate_config(filename, file,
                                                  "REVIEW"),
                         min_if_unless_guard: int=1,
                         indent_size: int=2,
-                        method_naming_convention: str='snake_case',
+                        method_naming_convention: str='snake',
                         string_literals: str='single_quotes',
-                        variable_naming_convention: str='snake_case',
+                        variable_naming_convention: str='snake',
```
Comment (sils) [inline bears/ruby/RuboCopBear.py]: hey you forgot to update the doc comment

**Comment by sils** [comment]
ack https://github.com/coala/coala-bears/pull/799/commits/71a9a317318558a9f4070da2d462cd361f0455fd

**Comment by sils** [comment]
@Asnelchristian rebase and merge please

**Comment by Asnelchristian** [comment]
reack https://github.com/coala/coala-bears/pull/799/commits/7fae4974ff29ba44162d88ab35cec5c0b350faca https://github.com/coala/coala-bears/pull/799/commits/4df9c20f78f6d9d81aaea3fac6e6355c743b7bd9

**Comment by Asnelchristian** [comment]
@rultor merge

**Pull request merged**